packages/core-commands/src/manage-reusable-blocks-command.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { useCommand } from '@wordpress/commands';
+import { __ } from '@wordpress/i18n';
+import { external } from '@wordpress/icons';
+
+export function useManageReusableBlocksCommand() {
```

Comment:
Nit: We could just name this useAdminNavigationCommands and place all potential WP-Admin links into this hook.

Reply:
Sounds good. I moved the add post/page hooks there too.